bin/select_gfa

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+
+import desimodel.focalplane
+import desitarget.io
+import fitsio
+import numpy as np
+import desimodel.io
+import desimodel.footprint
+import matplotlib.pyplot as plt
+import glob
+import time
+import os.path
+import argparse
+
+def near_tile(data, tilera, tiledec, window_ra=4.0, window_dec=4.0):
+    delta_RA = data['RA'] - tilera
```

Comment:
I would test this script against the `astropy.coordinates.SkyCoord` methods, which I think might be significantly faster (or maybe not!).

Reply:
I don't understand this. What outputs should be tested against what functions in `astropy.coordinates`? We are not doing coordinate transformations in the script.